src/main/java/org/terasology/signalling/componentSystem/SignalSwitchBehaviourSystem.java

```diff
@@ -106,6 +106,9 @@
 
     private Map<String, GateSignalChangeHandler> signalChangeHandlers = Maps.newHashMap();
 
+    /**
+     * Initialises the SignalSwitchBehaviorSystem's fields.
```

Comment:
So the trick about javadoc is finding a way to enhance knowledge beyond the names of involved methods, variables, etc. In this case since the name of the method is "initialise" that part is fairly clear. What isn't clear without looking at the code itself is what that entails :-)

In this case glancing over the code makes me think something like "Prepares related blocks and registers change handlers for when signals occur" would well describe what happens inside the method and educate the user beyond knowing some sort of initializing is happening